.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+## Description
+
+<!-- add description here -->
+
```

Comment:
We used to have some sort of guidance to open PRs. Some headers like `Context`, and/or `Changes`. We can bring that back.